content/config/otel.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Open Telemetry for the Docker CLI
+description: Learn about how you can leverage the Docker's capabilities to capture OTel traces
+keywords: otel, telemetry, traces, metrics, logs
+---
+
+{{< introduced engine 26.1.0 >}}
+
+The Docker Engine supports configurable [OTel](https://opentelemetry.io/docs/)
+utilities for tracing and monitoring. You can use these capabilities to capture
+information about `docker` CLI command invocations to get better insight into
+your Docker usage.
+
+## What is OTel?
+
+OTel stands for "Open Telemetry", and it's an open observability framework for
+creating and managing telemetry data, such as traces, metrics, and logs.
+OpenTelemetry is vendor- and tool-agnostic, meaning that it can be used with a
+broad variety of Observability backends.
+
+Support for OTel in the Docker CLI means that the CLI can emit information
+about events that take place, using the protocols and conventions defined in
+the Open Telemetry specification.
+
+## How it works
+
+The Docker CLI doesn't emit telemetry data by default. Only if you've seta an
```

Comment:
```suggestion
The Docker CLI doesn't emit telemetry data by default. Only if you've set an
```

Reply:
Where's your linter now, @dvdksn?

Reply:
i did this on purpose to make sure you read it